### ISSUE TITLE
Is this to BE a SAML IdP, or to build an app that interfaces with saml?

### ISSUE THREAD

**Issue by es-brookfield** [UWIT-IAM/nginx-saml-proxy#14]
Noobie question. I'm looking to connect a website I've built into my corporation's SAML 2.0 SSO workflow. My IT department has asked me to provide them with my SP & my ACS endpoint. If I clone the latest repo, I'd tell them "Then the SP entity-id to register is https://example.com/saml and the ACS endpoint to register is https://example.com/saml/login." Right?

Or do I have this all backwards - is this setup to make your own IdP to let other apps talk to it as the manager of all SAML login requests?

**Comment by EricHorst** [comment]
Yes, this setup makes a SP. You are correct about the entity-id and ACS endpoint.
